server/schema/deans.schema.ts

```diff
@@ -1,16 +1,10 @@
 import { relations, sql } from 'drizzle-orm';
-import {
-  integer,
-  pgTable,
-  smallserial,
-  text,
-  varchar,
-} from 'drizzle-orm/pg-core';
+import { integer, pgTable, serial, text, varchar } from 'drizzle-orm/pg-core';
 
 import { faculty } from '.';
 
 export const deans = pgTable('deans', {
-  id: smallserial('id').primaryKey(),
+  id: serial('id').primaryKey(),
```

Comment:
Why was this changed? Please don't change DB datatypes without prior discussion.

Reply:
I did rebase it though, i don't know how it didn't get changed back.